### ISSUE TITLE
files and grep to work with all workspace folders instead of only cwd

### ISSUE THREAD

**Issue by kuba-gaj** [neoclide/coc-lists#28]
coc.vim supports workspaces with multiple folders which is great future
(https://github.com/neoclide/coc.nvim/wiki/Using-workspaceFolders)

It would be amazing if files and grep list commands would work on all folders in the workspace instead of just current working directory.

**Comment by malob** [comment]
I would also love this. This is the main thing that is holding me back from switching from Denite.

**Comment by bkoropoff** [comment]
The files and grep lists should probably take the directory to search as an optional parameter.

**Comment by chemzqm** [comment]
Can be implemented by custom arguments :
* `--folders` for all workspace folders.
* `--folder` for workspace folder contains current buffer.

**Comment by laurent-george** [comment]
As fixed by @chemzqm in acf7c3d commit the options to use are:
  public options = [{
    name: '-F, -folder',
    description: 'Search files from current workspace folder instead of cwd.'
  }, {
    name: '-W, -workspace',
    description: 'Search files from all workspace folders instead of cwd.'
  }]

for instance use this in CocConfig:

    "list.source.grep.args": ["-workspace"]

**Comment by sotiwin** [comment]
It seems that files uses the args configuration differently. In case of files you have to set defaultArgs like this:
```javascript
"list.source.files.defaultArgs": ["--color", "never", "--files", "-workspace"],
```
